CREATE MODEL 1

Yeyo-GitHub opened the pull request:
MODEL 0 for 1.0
MODEL 1 for 1.0a

Yeyo-GitHub committed:
CREATE MODEL 1

MODEL 0 for 1.0
MODEL 1 for 1.0a

Yeyo-GitHub commented:
Hi Luc,

The bed in 1.0 (a) measures 200x200

The problem is the box cleaning, I must add the volume to be printed on the Surface of the bed. Since Repetier saw became well. Have I done wrong?

Then you propose, for example
  Model> 0 to 1.0 type 2.0 - 1 1.0 (a) 2.0 (a) ....
  type table> 1 for 1.0 2.0 - 2 for 1.0 (a) 2.0 (a)
??????

After your indication, the home pin 128, works right

luc-github commented:
Ok good if you can home at least you can use your printer

I suggest to use MODEL for pins difference between old and new generation
I suggest to use TYPE for difference between printer but not related to Generation like 1.0A and 2.0A

1.0 and 1.0A have a 200x200x200 build size  when 2.0 and 2.0A have 150x200x200 build size

I did not check the DEFINE in marlin for this but do not mix
X_MAX_POS which is the maximum value for X (X_MAX_POS=200)
from X_MAX_LENGTH which is a distance not a position => X_MAX_POS-X_MIN_POS

also you need to change X_MIN_POS, because at  home position, X is at -33, not 0, X_MIN_POS = -33
 same for Y

Yeyo-GitHub commented:
I agree, I will turn your suggestions.

I saw that e1_enable conflicts with z_homing, this pin is not defined in pinout.xls?

luc-github commented:
If you check well pinout.xls you will see it is a pin for 2.0, for 2.0A, extruder use another pin - I suggest you to refer to userpins.h of repetier it is more easy to follow the conditions

Yeyo-GitHub commented:
Sorry, it's true is beautifully displayed in userpins.h

Until now it was with pinout.xls.

Thanks :)

Yeyo-GitHub commented:
Luc, forgive the question but not if it has to do with the firmware:

In slic3r I have to rotate the piece to print oriented front.
So he did with repetier.
It may be firmware?

I want to adjust the menus, activate things missing.
You have suggestions?

thanks

luc-github commented:
object rotation is in slicer not in FW - FW just execute GCODE from slicer - I cannot say to much about slic3r I do not use it, sorry.

I suggest you to use the excel file to list what need to be changed first, it is visual and easy to change - 
https://github.com/luc-github/Marlin4Due/blob/master/Documentation/Menu%20Plans.xlsx, it is very convenient, I have used it for my repetier FW to display new menus,  
easy :  
https://cloud.githubusercontent.com/assets/8822552/4748170/bfa0b7e8-5a69-11e4-80b7-02b9c99fe122.png

and advanced:  
https://cloud.githubusercontent.com/assets/8822552/4748932/bebab9e2-5a7c-11e4-8fea-cdbe3d70820c.png

Yeyo-GitHub committed:
Creados tipos

Creado tipo de para dimensiones de cama 150*200 y 200*200

Ajustes probados sólo con 1.0a

Yeyo-GitHub committed:
ajustes

Activo el PWM para cama y ajusto pid incial

Yeyo-GitHub committed:
nEW BED PID

Yeyo-GitHub committed:
Ajuste pid
